translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+# Brazilian Portuguese translation.
+# Copyright (C) 2020
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR ctadeu@gmail.com, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: EMAIL\n"
+"POT-Creation-Date: 2017-03-14 21:32-0700\n"
+"PO-Revision-Date: 2020-10-22 17:08+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.4.1\n"
+"Last-Translator: Carlos Panato <ctadeu@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: pt_BR\n"
+"X-Poedit-KeywordsList: \n"
+
+#: pkg/kubectl/cmd/create_clusterrolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a ClusterRoleBinding for user1, user2, and group1 using the "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin --clusterrole=cluster-"
+"admin --user=user1 --user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t  # Criar o ClusterRoleBinding para user1, user2, e group1 utilizando o "
```

Comment:
Here I have to questions, just to define a standard for the translation:

1) Do we want the first verb to be "Criar" ou "Cria". 

Like "Cria um ClusterRoleBinding para user1, user2"

2) Because it's a literal translation of the sentence but not of the object name (thanks!!!), we can use "o ClusterRoleBinding", "a ClusterRoleBinding", "um ClusterRoleBinding" ou "uma ClusterRoleBinding".

I'm ok with any of them, just think it would be nice to have a standard/default for every sentence, to keep the consistency :)